app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "mv":
```

Comment:
The condition `len(parts) != 3 or parts[0] != "mv"` ensures the command format is correct. This is a good check, but consider adding a comment explaining this logic for clarity.